src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,129 @@
 package core.basesyntax;
 
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int DEFAULT_INITIAL_CAPACITY = 16;
+    private static final float DEFAULT_LOAD_FACTOR = 0.75f;
+    private static final int RESIZE_FACTOR = 2;
+
+    private Entry<K, V>[] table;
+    private int size;
+
+    public MyHashMap() {
+        this.table = new Entry[DEFAULT_INITIAL_CAPACITY];
```

Comment:
any reason for using keyword `this`?
```suggestion
        table = new Entry[DEFAULT_INITIAL_CAPACITY];
```